languages/csharp/exercises/concept/arrays/.docs/instructions.md

```diff
@@ -0,0 +1,57 @@
+You're an avid bird watcher that keeps track of how many birds have visited your garden in the last seven days.
+
+You have five tasks, all dealing with the numbers of birds that visited your garden.
+
+### 1. Calculate the total number of visiting birds
+
+Implement the `BirdCount.Total()` method to return the total number of birds that have visited your garden:
+
+```csharp
+int[] birdsPerDay = { 2, 5, 0, 7, 4, 1 };
+var birdCount = new BirdCount(birdsPerDay);
+birdCount.Total();
+// => 19
+```
+
+### 2. Calculate the number of busy days
+
+Some days are busier that others. A busy day is one where five or more birds have visited your garden.
+Implement the `BirdCount.BusyDays()` method to return the number of busy days:
+
+```csharp
+int[] birdsPerDay = { 2, 5, 0, 7, 4, 1 };
+var birdCount = new BirdCount(birdsPerDay);
+birdCount.BusyDays();
+// => 2
+```
+
+### 3. Check how many birds visited yesterday
+
+Implement the `BirdCount.Yesterday()` method to return how many birds visited your garden yesterday. The bird counts are ordered by day, with the first element being the count of six days ago, and the last element being today's count.
+
+```csharp
+int[] birdsPerDay = { 2, 5, 0, 7, 4, 1 };
+var birdCount = new BirdCount(birdsPerDay);
+birdCount.Yesterday();
+// => 4
+```
+
+### 4. Check if there was a day with no visiting birds
+
+Implement the `BirdCount.HasDayWithoutBirds()` method that returns `true` if there was a day at which no birds (count is zero) visited the garden; otherwise, return `false`:
```

Comment:
```suggestion
Implement the `BirdCount.HasDayWithoutBirds()` method that returns `true` if there was a day at which zero birds visited the garden; otherwise, return `false`:
```